identity-apps-core/components/org.wso2.identity.apps.common/src/test/java/org/wso2/identity/apps/common/listener/AppPortalRoleManagementListenerTest.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2023-2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
This should be 2024. Since this is a new file.
We have to update the year range only for the exiting files that we editing now

Reply:
This is now updated for the newly added AppPortalRoleManagementListenerTest.java and testng.xml files.